orangecontrib/text/widgets/owimportdocuments.py

```diff
@@ -635,10 +661,27 @@ def __onReportProgress(self, arg):
             self.pathlabel.setText(prettifypath(arg.lastpath))
             self.progress_widget.setValue(int(100 * arg.progress))
 
+    def add_features(self):
+        self.Warning.clear()
```

Comment:
Is clearing the warnings here intentional?

Reply:
Uh, it as probably left here from when there was a warning about missing lemmas (if the user selected only POS tags and not lemmas). Now we removed the warning and POS tags are loaded without lemmas - whatever that may mean for downstream preprocessing (leave it to the user). I'll remove the line.